Backend/src/main/java/com/onbank/api/dto/CreateTransferDto.java

```diff
@@ -0,0 +1,42 @@
+package com.onbank.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.onbank.api.model.OperationType;
+import lombok.Data;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Data
+public class CreateTransferDto {
+    @NotNull(message = "Date cannot be empty.")
+    @DateTimeFormat(iso = DateTimeFormat.ISO.TIME)
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private LocalDateTime date;
+    @NotNull(message = "Name cannot be empty.")
+    @Size(min = 1, max = 50, message = "Name must be between 1-50 characters.")
+    private String name;
+    @NotNull(message = "Surname cannot be empty.")
+    @Size(min = 1, max = 100, message = "Surname must be between 1-100 characters.")
+    private String surname;
+    @NotNull(message = "Account number cannot be empty.")
+    @Size(min = 26, max = 28, message = "Account number must be between 26-28 characters.")
+    @Pattern(regexp = "(^[A-Z]{2}\\d{26}$)|(^\\d{26}$)",
+            message = "Invalid account number.")
+    private String accountNumber;
```

Comment:
Myślę że to jest dobry moment żeby zamockować numer nadawcy, imię i nazwisko nadawcy. Bo obecnie mamy tylko informację o odbiorcy, a transfer to operacja dwustronna w sumie :P

Reply:
A kiedy już dodamy do transaction numer nadawcy, to możemy pobierać listę transakcji dla danego numeru, zamiast listy wszystkich (lekko zmodyfikować). Bo obecnie nam wszystkie transakcje wypisze dla wszystkich kont. Dodatkowo przy pobieraniu transakcji trzeba jakoś odróżniać które są obciążeniami, a które są uznaniami. Myślę że fajnie by było sobie przegadać te tematy jutro na spokojnie i ustalić co i jak, żeby to miało ręce i nogi ;)